### PR TITLE
Legend documentation + small css fix

### PR DESCRIPTION
**Pull request by bjedrzejewski**
Simple legend documentation page, based on the example from the visual tests. Small css fix for using legend component on the site.

### CODE REVIEW

File: site/src/components/chart/legend.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+          updateField(datum, 'low');
+  
+          d3.select('#legend')
+              .data([datum])
+              .call(legend);
+      }
+  
+      renderLegend();
+      setInterval(function () {
+          renderLegend();
```
Comment (chrisprice) [inline site/src/components/chart/legend.md]: Is this equivalent to -
```js
setInterval(renderLegend, 1000)

**Comment by chrisprice** [comment]
Waiting for an answer on https://github.com/ScottLogic/d3fc/pull/633#discussion_r42383668 before merging.

File: site/src/components/chart/legend.md (hunk 1 above)
Comment (bjedrzejewski) [inline site/src/components/chart/legend.md]: Thanks for spotting, simplified.

**Comment by chrisprice** [comment]
Thanks @bjedrzejewski, as this isn't a functional change I'm not going to perform a release.

**Pull request merged**